.github/workflows/compliance.yaml

```diff
@@ -0,0 +1,58 @@
+name: Compliance
+
+on:
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - synchronize
+      - reopened
+      - unassigned
+
+jobs:
+  check-for-linked-issue:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Check for keyword and issue number
+        id: check-for-keyword
+        uses: actions/github-script@v7
+        with:
+          script: |
```

Comment:
nit. We could try adding some more whitespace whitespace between logical blocks of code to help improve readability.

Reply:
Hey @glenflorendo thanks for comment. Without reviewing the specific code (yet), are you referring to adding linebreaks such as shown here:

<details>
<img width="698" alt="Screenshot 2024-04-02 112224" src="https://github.com/hackforla/lucky-parking/assets/40799239/6b96c137-3909-44cd-b786-5103323a1364">

</details>

Reply:
Yes, good job!